tools/build/Makefile-common.in

```diff
@@ -81,14 +81,22 @@ COMMON_NQP_SOURCES  = \
 
 NQP_COMBINED = NQP.nqp
 
+// No QAST_SOURCES, just one file, no need to combine
```

Comment:
It does not. I would be surprised if it worked with any make implementation.
That's the embarrassing syntax mistake I talked about - it's fixed now ;-)